### ISSUE TITLE
Accessibility: Colour contrast

### ISSUE THREAD

**Issue by sindicatoesp** [PreventionWeb/delta#29]
---
Source: https://github.com/unisdr/DELTA/issues/29
Source author: @andersbhansen
Source created: 2025-01-13T08:43:26Z

**Comment by sindicatoesp** [comment]
[Migrated comment from unisdr/DELTA#29]
Original author: @robertopanzeri
Original created: 2025-01-13T08:56:07Z

Failed color contrast:

- https://rawgit.com/PreventionWeb/templates/dts/dts/dist/base.html
  - White text over light blue bkg (UNDRR footer button hover status)
- https://rawgit.com/PreventionWeb/templates/dts/dts/dist/form.html
  - UNDRR Teal text over white bkg (link) and viceversa (secondary button)
  - UNDRR Teal text over yellow bkg (link over warning alert)
  - Blue text over white bkg (outline button hover status)
  - White text over light blue bkg (primary button hover status)
- https://rawgit.com/PreventionWeb/templates/dts/dts/dist/add-edit-filter.html
  - UNDRR Teal text over grey bkg (link over highlighted metadata)
  - Blue text over white bkg (system button hover status)
  - Colored dot over white bkg (all status indicators except for Published)
  - Blue text over white bkg (ghost button)
- https://rawgit.com/PreventionWeb/templates/dts/dts/dist/add-record.html
  - UNDRR Teal text over grey bkg (link over form section)
- https://rawgit.com/PreventionWeb/templates/dts/dts/dist/analytics-sectors.html
  - Grey text over grey bkg (map tabs)

**Comment by sindicatoesp** [comment]
[Migrated comment from unisdr/DELTA#29]
Original author: @robertopanzeri
Original created: 2025-01-13T08:56:38Z

Color contrast fixes:

- **Primary button**
  - Hover text color from #FFFFFF (mg-color-neutral-0) to #181823 (dts-coal-600)
- **Secondary button**
  - Default bkg color from #00AFAE (dts-iris-300) to #007B7A (dts-iris-400)
  - Hover text color from #FFFFFF (mg-color-neutral-0) to #181823 (dts-coal-600)
- **Ghost button**
  - Default text color from #6093BD (dts-cerulean-300) to #106CB8 (dts-cerulean-400)
  - Hover bkg color from #6093BD (dts-cerulean-300) to #106CB8 (dts-cerulean-400)
- **Outline button**
  - Hover bkg color from #FFFFFF (mg-color-neutral-0) to #106CB8 (dts-cerulean-400)
  - Hover border color from #3372a7 (mg-color-blue-700) to #106CB8 (dts-cerulean-400)
  - Hover text color from #3372a7 (mg-color-blue-700) to #FFFFFF (mg-color-neutral-0)
- **System button**
  - Hover bkg color from #FFFFFF (mg-color-neutral-0) to #106CB8 (dts-cerulean-400)
  - Hover border color from #3372a7 (mg-color-blue-700) to #106CB8 (dts-cerulean-400)
  - Hover text color from #3372a7 (mg-color-blue-700) to #FFFFFF (mg-color-neutral-0)
- **UNDRR footer button**
  - Hover text color from #FFFFFF (mg-color-neutral-0) to #181823 (dts-coal-600)
- **Links**
  - Text color from #00AFAE (dts-iris-300) to #007B7A (dts-iris-400)
- **Metadata highlight box** (https://rawgit.com/PreventionWeb/templates/dts/dts/dist/add-edit-filter.html)
  - Bkg color from #DDE8F0 (dts-cerulean-100) to #F2F2F2 (dts-coal-100)
- **Map tabs**
  - Default bkg color from #F2F2F2 (dts-coal-100) to #FFFFFF (mg-color-neutral-0)
  - Text color from #979797 (dts-coal-400) to #333333 (dts-grey-500)
- **Status**
  - Draft bkg color from #F4E496 (dts-vis-vis) to #FFFFFF (mg-color-neutral-0) with border #181823 (dts-coal-600)
  - Waiting for information bkg color from #BCAE67 (dts-gimblet) to #D87838 (dts-tango-600)
  - Waiting for validation bkg color from #FEC124 (dts-turbo) to #AD66A1 (dts-violet-500)
  - Published bkg color from #026666 (dts-teal-700) to #6D9A75 (dts-killarney-500)
  - Published to UNDRR bkg color from #9ABBD6 (dts-cerulean-200) to #106CB8 (dts-cerulean-400)
